### DISCUSSION TITLE
Cycle plan until 2023-09-14

### DISCUSSION THREAD

**Discussion by onOffice-Web** [onOffice-Web-Org/oo-wp-plugin#617] [Announcements]
For the next cycle please work on [milestone v4.16](https://github.com/onOffice-Web-Org/oo-wp-plugin/milestone/5)

- https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/619 ⚠High priority⚠
- https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/550 ⚠High priority⚠
- https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/581
- https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/607
- https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/616
- https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/613 (on hold for now)

The cycle period is until Tue 2023-09-14, inclusive.

For any questions, please don't hesitate to ask.

**Comment by yeneastgate** [comment]
@fredericalpers I will focus that high priority task, now. :D
